keyboards/yandrstudio/sp60/rev2/rev2.c

```diff
@@ -0,0 +1,8 @@
+// Copyright 2022 Y&R-Biu (@jiaxin96)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "rev2.h"
```

Comment:
```suggestion
#include "quantum.h"
```
After deleting the .h file.